bluepyemodel/access_point/local.py

```diff
@@ -789,3 +789,30 @@ def add_entry_recipes(
 
         with recipes_path.open("w") as f:
             json.dump(recipes, f, indent=2)
+
+    def store_hocs(
+        self,
+        only_validated=False,
+        only_best=True,
+        seeds=None,
+        map_function=map,
+        new_emodel_name=None,
+        description=None,
+        output_base_dir="export_emodels_hoc",
+    ):
+        pass
```

Comment:
Wouldn't it be better to raise a NotImplementedError instead of using 'pass' for those functions for clarity and debugging?

Reply:
done in latest commit